torch/fx/experimental/test_rewriter.py

```diff
@@ -0,0 +1,51 @@
+import torch
```

Comment:
Can you move this file to `test/` or move these test cases to `test/test_fx_experimental.py`?